starknet_py/net/account/account_test.py

```diff
@@ -62,13 +62,14 @@ def test_create_account_from_signer():
     assert account.signer == signer
 
 
-def test_create_account_raises_on_no_chain_and_signer():
```

Comment:
So i get that chain can be somehow constructed from the `net` even if signer is not provided, but what if non-standard net is used and chain creation fails? We should test for a case like this.